objects/Ranomizer/Draw_64.gml

```diff
@@ -28,12 +28,15 @@ else
 	alarm_1_trigger = 0
 }
 
-
-	//draw_set_halign(fa_center);
-	//draw_set_valign(fa_center);
-	//draw_set_font(titleFont);
-	//draw_text(room_width/2, top_buffer, crazy_level);	
-	//draw_set_font(scoreFont);
+if (oPlayer.testing = 1)
```

Comment:
```suggestion
if (oPlayer.testing == 1)
```

Note that a single `=` is assignment, while `==` tests equality of two statments